### PR TITLE
[DDoS] Add managed ruleset change log pages

### PR DESCRIPTION
**Pull request by jdesgats**
There is nothing yet, first public changes will come later.

### CODE REVIEW

File: content/ddos-protection/change-log/_index.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+title: Change log
+pcx-content-type: concept
+weight: 20
+meta:
+  title: Change log for DDoS Managed Rulesets
+---
+
+# Change log for DDoS Managed Rulesets
+
+Cloudflare has a regular cadence of releasing updates and new rules to the DDoS maangeed rulesets. The updates either improve a rule's accuracy, lower false positives rates or increase the protection due to a change in the threat landscape.
```
Comment (pedrosousa) [inline content/ddos-protection/change-log/_index.md]:
```suggestion
Cloudflare has a regular cadence of releasing updates and new rules to the DDoS Managed Rulesets. The updates either improve a rule's accuracy, lower false positives rates, or increase the protection due to a change in the threat landscape.
```

File: content/ddos-protection/change-log/l7/_index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: HTTP DDoS Managed Rules change log
```
Comment (pedrosousa) [inline content/ddos-protection/change-log/l7/_index.md]:
```suggestion
title: HTTP DDoS Managed Rules
```
In the context of the left sidebar, we can make the TOC entry text shorter.

File: content/ddos-protection/change-log/l4/_index.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
+title: Network-layer DDoS Managed Rules change log
```
Comment (pedrosousa) [inline content/ddos-protection/change-log/l4/_index.md]:
```suggestion
title: Network-layer DDoS Managed Rules
```
In the context of the left sidebar, we can make the TOC entry text shorter (and we're already in the Change log section).

File: content/ddos-protection/change-log/l7/_index.md (hunk 2 above)
Comment (jdesgats) [reply content/ddos-protection/change-log/l7/_index.md]: Applied

File: content/ddos-protection/change-log/l4/_index.md (hunk 3 above)
Comment (jdesgats) [reply content/ddos-protection/change-log/l4/_index.md]: Applied

**Review by CryptoGodfatherVA23 — APPROVED** [review]

**Review by pedrosousa — APPROVED** [review]

**Pull request merged**